File: macos/GladIs-macOS/PDFViewerManager.m
//
//  PDFViewerManager.m
//  GladIs-macOS
//
//  Created by Raphaël Payet on 21/02/2024.
//

#import <Foundation/Foundation.h>
#import "React/RCTViewManager.h"

@interface RCT_EXTERN_MODULE(PDFViewerManager, RCTViewManager)

RCT_EXPORT_VIEW_PROPERTY(url, NSString)

@end

File: macos/GladIs-macOS/PDFViewerManager.swift
//
//  PDFViewerManager.swift
//  GladIs-macOS
//
//  Created by Raphaël Payet on 21/02/2024.
//

import Foundation
import React

@objc(PDFViewerManager)
class PDFViewerManager: RCTViewManager {
  
  override func view() -> NSView! {
      return PDFViewer()
  }

  override static func requiresMainQueueSetup() -> Bool {
      return true
  }
}
